src/www/ui/api/Controllers/JobController.php

```diff
@@ -172,6 +174,29 @@ public function createJob($request, $response, $args)
       return $response->withJson($error->getArray(), $error->getCode());
     }
   }
+  /**
+   * Get all jobs created by the current user.
+   *
+   * @param integer|null $id Specific job id or null for all jobs
+   * @param integer $uid Specific user id
+   * @param ResponseHelper $response Response object
+   * @param integer $limit   Limit of jobs per page
+   * @param integer $page    Page number required
+   * @return ResponseHelper
+   */
+  private function getAllUserResults($id,$uid, $response, $limit, $page)
+  {
+    list($jobs, $count) = $this->dbHelper->getUserJobs($id,$uid, $limit, $page);
```

Comment:
```suggestion
  private function getAllUserResults($id, $uid, $response, $limit, $page)
  {
    list($jobs, $count) = $this->dbHelper->getUserJobs($id, $uid, $limit, $page);
```

Reply:
We are using [PHPCS](https://github.com/squizlabs/PHP_CodeSniffer) to do it. There should be plugins for your IDE as well.